<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>SubscribeWengo2</class>
 <widget class="QWidget" name="SubscribeWengo2" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>329</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>WengoPhone - Wengo Account Creation (2/2)</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="3" column="0" >
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
    </spacer>
   </item>
   <item row="2" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="1" >
      <widget class="QLineEdit" name="nicknameLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1" >
      <widget class="QLineEdit" name="emailLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0" >
      <widget class="QLabel" name="nicknameLabel" >
       <property name="text" >
        <string>Nickname:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>Email:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" >
      <widget class="QLineEdit" name="passwordLineEdit" >
       <property name="enabled" >
        <bool>false</bool>
       </property>
       <property name="echoMode" >
        <enum>QLineEdit::Password</enum>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="passwordLabel" >
       <property name="text" >
        <string>Password:</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" >
    <widget class="QLabel" name="label_2" >
     <property name="text" >
      <string>Please note your details needed to access your account:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" >
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item row="0" column="0" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" >
      <widget class="QPushButton" name="finishButton" >
       <property name="text" >
        <string>Finish</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" >
      <widget class="QPushButton" name="addIMAccountButton" >
       <property name="text" >
        <string>Add your IM Accounts to your Profile</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>Contratulations!

Your account has been successfully created</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>finishButton</tabstop>
 </tabstops>
 <resources>
  <include location="../wengophone.qrc" />
  <include location="../../wengophone.qrc" />
 </resources>
 <connections/>
</ui>
